src/packages/@ncigdc/components/__tests__/FacetWrapper.tests.js

```diff
@@ -0,0 +1,17 @@
+import { getFacetType } from '@ncigdc/components/FacetWrapper';
+
+describe('getFacetType', () => {
+  describe('returns "range"', () => {
```

Comment:
We could test returning other facet types; otoh we had no tests at all before.

Reply:
I had contemplated adding more tests, but that code has some oddities to it. Some of it appears to be outdated and the order of the if blocks appear to matter.  This makes the intent of this function a little ambiguous.  Due to that, I didn't want to write tests for the existing behavior that I couldn't verify as correct and limited the tests to the scope of the bug.